### PR TITLE
[MI-3774] Update info page to make it mobile responsive

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
Update info page to make it mobile responsive

#### Screenshots
image
image
image
image
image
image

### CODE REVIEW

**Review by raghavaggarwal2308 — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Comment by avas27JTG** [comment]
@ayusht2810 If selecting any one of the options is mandatory, I think the "Continue" button should be disabled until an option is selected.

**Review by larkox — APPROVED** [review]
No new comments from my side.

File: assets/info-page/index.html [hunk 1]
```diff
@@ -1,39 +1,69 @@
...
+      const msteamsCardBody = 'msteams-card__body';
+      const mmCardHeading = 'mm-card__heading';
+      const msteamsCardHeading = 'msteams-card__heading';
+      const selectButtonClass = 'selection-button';
+      let choosenPlatform = '';
 
-      function closeInfoPage() {
+      closeInfoPage = () => {
         window.close();
       }
```
Comment (avas27JTG) [inline assets/info-page/index.html:20]:
```suggestion
      closeInfoPage = () => window.close();
```

**Review by avas27JTG — APPROVED** [review]

**Comment by lieut-data** [comment]
The current page has since been updated, and we'll iterate from there as needed.

**Pull request closed without merging**